qiskit/circuit/library/__init__.py

```diff
@@ -131,36 +191,95 @@
    XGate
    YGate
    ZGate
+
+2-qubit standard gates
+----------------------
+
+.. autosummary::
+   :toctree: ../stubs/
+
+   CHGate
+   CPhaseGate
+   CRXGate
+   CRYGate
+   CRZGate
+   CSGate
+   CSdgGate
+   CSXGate
+   CUGate
+   CU1Gate
+   CU3Gate
+   CXGate
+   CYGate
+   CZGate
+   DCXGate
+   ECRGate
+   iSwapGate
+   RCCXGate
+   RXXGate
+   RYYGate
+   RZXGate
+   RZZGate
+   SwapGate
+   XXMinusYYGate
+   XXPlusYYGate
+
+3+ qubit standard gates
+-----------------------
+
+.. autosummary::
+   :toctree: ../stubs/
+
+   C3SXGate
+   C3XGate
+   C4XGate
+   CCXGate
+   CCZGate
+   CSwapGate
+   RC3XGate
+
+Global standard gate
+--------------------
```

Comment:
For consistency in the titles (although there's only 1):
```suggestion
Global standard gates
---------------------
```